connector/elasticconnector/connector.go

```diff
@@ -4,8 +4,8 @@ import (
 	"context"
 	"strings"
 
-	"github.com/tommyers-elastic/opentelemetry-collector-contrib/connector/elasticconnector/internal/datastream"
-	"github.com/tommyers-elastic/opentelemetry-collector-contrib/connector/elasticconnector/internal/hostmetrics"
+	"github.com/ishleenk17/opentelemetry-collector-contrib/connector/elasticconnector/internal/datastream"
```

Comment:
please revert these import changes

Reply:
you can get around the naming issue by using a `replace`, there is an example in the config already.

Reply:
Ok, I'll try it this way